EventFilter/CTPPSRawToDigi/plugins/CTPPSPixelRawToDigi.cc

```diff
@@ -35,8 +35,14 @@ CTPPSPixelRawToDigi::CTPPSPixelRawToDigi( const edm::ParameterSet& conf )
   produces< edm::DetSetVector<CTPPSPixelDigi> >();
 
 //CablingMap could have a label //Tav
-  mappingLabel_ = config_.getParameter<std::string> ("mappingLabel"); //RPix
+  includeErrors_ = config_.getParameter<bool> ("IncludeErrors");
```

Comment:
capitalization style applies to python parameters as well.
In any case, it's best to be at least consistent (lower case in mappingLabel vs upper case in IncludeErrors)

Reply:
done